contracts/src/SequencerInbox.sol

```diff
@@ -90,6 +92,9 @@ contract SequencerInbox is ISequencerInbox, Initializable {
                 }
                 runningAccumulator = keccak256(abi.encodePacked(runningAccumulator, numTxs, prefixHash, txDataHash));
                 dataOffset += txLength;
+                if ((dataOffset - initialDataOffset) > txBatch.length) {
+                    revert TxDataOverflow();
```

Comment:
need to define this error in `ISequencerInbox`

Reply:
Done. However, I have moved this patching stuff to a different PR, so that testing PR and bug fix PR becomes different. Please check #46